particles/GiftList/GiftList.ptcl

```diff
@@ -8,6 +8,6 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-GiftList(in Person person, in [Product] list, out [Product] resultList)
+GiftList(in Person person)
 
-renders: root(list) none-need
\ No newline at end of file
+renders: preamble(list) none-need
```

Comment:
alternatively, i think, maybe you could not specify the view for which the slot is exposed in ListView.ptcl:
exposes: action(list), preamble
and then you don't need to specify here the view that is being rendered (because it is not). here is an example for this in Shane's demo PR:
https://github.com/PolymerLabs/arcs/pull/160/files#diff-b5f5c110e5e1ac231ca810d2335afe63